CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Bug where an empty destination list in a device's backup set broke creation of DeviceSettings objects for that device.
 
+### Added
+
+- Added new exception `Py42UserDoesNotExist` to throw error if `remove()` throws `400` when user does not exist in the detection list.
```

Comment:
We should call this exception something else

Reply:
Changed.